eng/scripts/Fetch-Namespaces-DotnetNupkg.ps1

```diff
@@ -0,0 +1,44 @@
+<#
+  .SYNOPSIS
+  Fetch the namespaces from dotnet nupkg and checkin to docs/metadata
+
+  .DESCRIPTION
+  The scripts is used for docs.ms to fetch namespace list for particular nupkg artifact.
+
+  .PARAMETER DotnetNupkgLocations
+  Specifies the fo nupkg location.
+
+  .PARAMETER DocRepoLocation
+  Specifies Location of the root of the docs.microsoft.com reference doc location. 
+#>
+[CmdletBinding()]
+param (
+  [string] $DotnetNupkgLocations,
+  [string] $DocRepoLocation
+)
+
+Set-StrictMode -Version 3
+. (Join-Path $PSScriptRoot ../common/scripts/common.ps1)
+$ErrorActionPreference = 'Continue'
+
+$nupkgInfos = @($DotnetNupkgLocations | ForEach-Object { GetPackageInfoJson $_ })
+foreach($nupkgLocation in $nupkgInfos) {
+  $artifactName = Split-Path $nupkgLocation -Leaf
+  Write-Host "Nupkg Location: $nupkgLocation"
+  Write-Host "DocRepoLocation: $DocRepoLocation"
+  
+  $nupkgFile = Get-dotnet-Package-Artifacts -Location "$nupkgLocation"
```

Comment:
We should consider using Get-dotnet-PackageInfoFromPackageFile instead which also extracts the file and we can do work to extract the namespaces as part of that step.